### ISSUE TITLE
How to use ground truth polygons for training

### ISSUE THREAD

**Issue by avisekiit** [RodinDmitry/glare#2]
Dear Authors,

Thanks for sharing the dataset. If I understand, the markup xml files contain bounding polygons for each image. 
But I was not sure how you are using the polygons while training the network?
Do you create another binary image with label = 1 inside the polygon regions else 0 ?

By seeing your architecture, I see that you output 1 channel as output from the deep net.
So, are you doing some kind of dense prediction ?
It will be really helpful to know the loss function (what is predicted and what is the expected output) used for training the network.

Thanks,

Avisek
Indian Institute of Technology Kharagpur
